packages/millicast-sdk/src/Publish.js

```diff
@@ -110,7 +111,7 @@ export default class Publish extends BaseWebRTC {
       logger.error('Error while broadcasting. Publisher data required')
       throw new Error('Publisher data required')
     }
-    const recordingAvailable = jwtDecode(publisherData.jwt).millicast.record
+    const recordingAvailable = jwtDecode(publisherData.jwt)[atob('bWlsbGljYXN0')].record
```

Comment:
Is there any reason to decode the base64-encoded string 'millicast' and use it as a key on the decoded JWT? To me this looks like unnecessary complexity for a single value lookup

Reply:
@nicolaslevy